common/lib/xmodule/xmodule/js/src/video/00_sjson.js

```diff
@@ -51,11 +67,38 @@ function() {
             return min;
         };
 
+        var filter = function (start, end) {
+            var filteredTimes = [],
+                filteredCaptions = [],
+                startTimes = getStartTimes(),
+                captions = getCaptions(),
+                cur,
+                i;
+
+            if (startTimes.length != captions.length) {
```

Comment:
reference for the above statement: http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons#answer-359509